server/models/dataset_helpers/image_metadata.py

```diff
@@ -737,15 +741,31 @@ def _populateMetadata(acquisition, clinical):
 
     # Set melanocytic field based on diagnosis
     if diagnosis in MelanocyticFieldParser.melanocyticDiagnoses:
-        if clinical.get('melanocytic') is None:
+        melanocytic = clinical.get('melanocytic')
```

Comment:
Is there a reason not to put this statement outside the conditional, so it can be shared by both branches?

Reply:
Fixed.